### PR TITLE
Add Feature Route Domain in CRD Mode

### PR DESCRIPTION
**Pull request by nandakishorepeddi**
**Description**:  _Add Issue/Feature description_
CIS build : nandakishoref5/k8s-bigip-ctlr:rd_v2
**Changes Proposed in PR**:

**Fixes**: #_Github issue id_

## General Checklist
- [x] Updated Added functionality/ bug fix in release notes
- [x] Added examples for new feature
- [x] Updated the documentation

## CRD Checklist
- [x] Updated required CR schema

### CODE REVIEW

File: cmd/k8s-bigip-ctlr/main.go [hunk 1]
```diff
@@ -222,6 +223,8 @@ func _init() {
 	// Custom Resource
 	customResourceMode = globalFlags.Bool("custom-resource-mode", false,
 		"Optional, When set to true, controller processes only F5 Custom Resources.")
+	defaultRouteDomain = globalFlags.Int("default-route-domain", 0,
+		"Optional, When set to true, controller processes only F5 Custom Resources.")
```
Comment (agupta49) [inline cmd/k8s-bigip-ctlr/main.go:227]: This message should be appropriate.

File: pkg/crmanager/types.go [hunk 2]
```diff
@@ -180,6 +182,7 @@ type (
 		ServiceAddress []ServiceAddress `json:"serviceAddress,omitempty"`
 		IRulesMap      IRulesMap
 		IntDgMap       InternalDataGroupMap
+
```
Comment (subbuv26) [inline pkg/crmanager/types.go:185]: need not put this extra line

File: cmd/k8s-bigip-ctlr/main.go [hunk 3]
```diff
@@ -224,7 +224,7 @@ func _init() {
 	customResourceMode = globalFlags.Bool("custom-resource-mode", false,
 		"Optional, When set to true, controller processes only F5 Custom Resources.")
 	defaultRouteDomain = globalFlags.Int("default-route-domain", 0,
-		"Optional, When set to true, controller processes only F5 Custom Resources.")
+		"Optional, CIS uses this value while configuring AS3 Declaration with respective route domain ")
```
Comment (trinaths) [inline cmd/k8s-bigip-ctlr/main.go:227]: Provide partition default route domain ID. Default ID is 0.

**Comment by trinaths** [comment]
please set the Github issue, https://github.com/F5Networks/k8s-bigip-ctlr/issues/1730

File: cmd/k8s-bigip-ctlr/main.go (hunk 3 above)
Comment (nandakishorepeddi) [reply cmd/k8s-bigip-ctlr/main.go:227]: "default-route-domain", 0, here 0 represents default RD ID.

**Comment by nandakishorepeddi** [comment]
Closing this PR as review comments are addressed in  https://github.com/F5Networks/k8s-bigip-ctlr/pull/1779

**Pull request closed without merging**